app/modules/course/views/CoursesView.js

```diff
@@ -23,19 +23,19 @@ define(function(require) {
         },
 
         beforeRender: function() {
-            this.renderList();
+            this.renderList();  /*
```

Comment:
I have reverted the original state of file "CoursesView.js" after my experiments